metricbeat/module/logstash/_meta/supported-versions.yml

```diff
@@ -0,0 +1,3 @@
+variants:
+  - LOGSTASH_VERSION: 8.1.0-aa69d697-SNAPSHOT
+  - LOGSTASH_VERSION: 7.17.0-fd761817-SNAPSHOT
```

Comment:
Consider including some released version too, to ensure compatibility with older versions, and to have a reference to compare with if upgrading a snapshot makes the test fail.

Reply:
Yeah, makes sense. For now I'll add 7.17 and 8.0.0-rc2 I guess (cc @robbavey for cross check).

I have no idea how we'll keep those updated but maybe @v1v has some ideas and we have some time to figure it out.

Reply:
The special case here are the snapshot versions, so I think we should have something like the bump stack versions approach.

The non-snapshot versions can be maintained manually, as done for the rest of modules, when we find some particularity in an specific version, or when we consider that we don't need testing for an specific versions anymore.